modules/appeals_api/app/services/appeals_api/supporting_evidence/temporary_storage_uploader.rb

```diff
@@ -1,44 +0,0 @@
-# frozen_string_literal: true
```

Comment:
Same here. Let's keep this and let `upload_complete` decide if it want so use them or not.